sqlparser/parse_test.go

```diff
@@ -1577,6 +1577,8 @@ func TestConvert(t *testing.T) {
 		input: "select convert('abc', json) from t",
 	}, {
 		input: "select convert('abc' using ascii) from t",
+	}, {
+		input: "select $1::integer::bytea from dual where test = $2::varchar::bytea",
```

Comment:
`$1::integer::bytea` - this is valid only in postgres, right?

Reply:
yes, only postgres